libraries/AP_Gripper/AP_Gripper_EPM.cpp

```diff
@@ -35,6 +35,8 @@ void AP_Gripper_EPM::init_gripper()
 
     // initialise the EPM to the neutral position
     neutral();
+    // flag we are in the neutral position
+    config.state = AP_Gripper::STATE_NEUTRAL;
```

Comment:
Isn't this already done in the header?

Reply:
Yes, does that mean I should remove it from the init or is it fine to leave it here for clarity?

Reply:
Remove - it costs us unnecessary bytes.

Reply:
Thanks Peter, done.